.gitignore

```diff
@@ -1,3 +1,4 @@
+built/
```

Comment:
Small change: I pretty much use `build` or `dist` everywhere, could you use one of those?

Reply:
Absolutely!